Classes/Settings.cs

```diff
@@ -369,6 +330,36 @@ internal static int RssServerNumRecentEps
             }
         }
 
+        internal static string SetSaveFolder(Model.Programme progInfo)
```

Comment:
"Should work like properties", but unfortunately this isn't possible, is that what you meant? I suppose we agree about that functions should be differentiated by Get and Set.